ambry-api/src/main/java/com.github.ambry/router/GetBlobOptions.java

```diff
@@ -28,6 +28,8 @@
   private final ByteRange range;
   // Flag indicating whether to return the raw blob payload without deserialization.
   private final boolean rawMode;
+  private final long blobSegment;
```

Comment:
This can be an int.  We aren't realistically going to have billions of segments in a blob.

Reply:
yeah it should be an int, I think it got turned into a long when conflated with the blob byte size params, which are longs